core/src/ledger_cleanup_service.rs

```diff
@@ -17,6 +17,8 @@ use std::time::Duration;
 // - To make sure that if a validator needs to reboot from its own snapshot, it has enough slots locally
 //   to catch back up to where it was when it stopped
 pub const DEFAULT_MAX_LEDGER_SLOTS: u64 = 270_000;
+// Gives about half an hour of slots to rollback from.
+pub const DEFAULT_MIN_LEDGER_SLOTS: u64 = 1_000;
```

Comment:
Can we use the existing DEFAULT_MAX_LEDGER_SLOTS?

Reply:
Like set the default to 1,000? or revert this part of the change entirely? I think 1,000 is too low for the default, but I would allow people to set it this low if they are really hurting for disk space, didn't have much stake and had high risk tolerance.

Reply:
270_000, DEFAULT_MAX_LEDGER_SLOTS, was the previous minimum.  This gives a couple hours of slots to serve to other nodes that are booting and seems to be working pretty good.

Reply:
That's 1.5 days of slots.

Reply:
If the comment is correct about 100 slots being 2gb, then 270,000 at 50k tps is 5TB of disk space.

Reply:
But I don't like that it's delineated in slots, if the slots are full, you need to purge faster, if they are mostly empty, you can wait and purge and keep more data to recover from if needed.

Reply:
We're not going to be 50k TPS for quite while I suspect, but yes totally right.   For the next couple months, I'll be ecstatic with rock solid idling

> But I don't like that it's delineated in slots, if the slots are full, you need to purge faster, if they are mostly empty, you can wait and purge and keep more data to recover from if needed.

Yeah I agree.  It would be nice to improve that sometime.  I don't like that either